src/index.css

```diff
@@ -49,3 +49,21 @@ body,
 .customized-scrollbar::-webkit-scrollbar-track {
   background: transparent;
 }
+
+.rounded-xl {
+  border-radius: 1rem;
+}
+
+.rounded-b-xl {
+  border-bottom-right-radius: 1rem;
+  border-bottom-left-radius: 1rem;
+}
+
+.rounded-3xl {
+  border-radius: 1.75rem;
+}
+
+.bg-indigo-50 {
+  --tw-bg-opacity: 1;
+  background-color: rgba(247, 245, 255, var(--tw-bg-opacity));
+}
```

Comment:
Tailwind CSS includes this utility class names (see the [documentation](https://tailwindcss.com/docs/border-radius)). Why do you declare them again?

Reply:
I don't know why but, it's just not working if don't declare rounded-xl/3xl/b-xl myself. I'm also confused about this TvT